libxcb.rb

```diff
@@ -14,10 +14,11 @@ class Libxcb < Formula
   option "with-static", "Build static libraries (not recommended)"
   option "with-docs",   "Generate API documentation"
 
-  depends_on "pkg-config"  => :build
+  depends_on "pkg-config"       => :build
```

Comment:
Please don't line up the `=>` operators.

I realize that a lot of the existing code does do do that, but I would prefer it if it didn't. My apologies - I hope it didn't take too long to do!